MiniSim/MenuItems/SubMenuItem.swift

```diff
@@ -119,6 +119,18 @@ enum SubMenuItems {
             accessibilityDescription: "Delete simulator"
         )
     }
+    
+    struct DeleteEmulator: SubMenuActionItem {
+        let title = NSLocalizedString("Delete Emulator", comment: "")
```

Comment:
Let's use lower case for `Emulator`. So `Delete emulator`

Reply:
Ok

Reply:
@okwasniewski  I have updated the PR as per your review. Please check